### ISSUE TITLE
find_similar page is slow to respond

### ISSUE THREAD

**Issue by chrisfilo** [NeuroVault/NeuroVault#147]
It takes over 8s for the server to reply. Optimisation can include:
* better queries (doing sorting  and picking to 100 in the query, querying for values instead of objects)
* better indexing
* looking to pycompare performance

**Comment by chrisfilo** [comment]
part of the problem is that it loads a different version of jquery from a external site

**Comment by vsoch** [comment]
This issue will be addressed by [this commit](https://github.com/vsoch/pybraincompare/commit/6c54cc29b5723c4d81a324d5c33cdfa042e17b46)

I've added a "remove_scripts" variable to the function that will allow the user to specify removing a resource (eg, javascript or css script) from the template, in the case that he/she is embedding in a page that already has some other version (such as NeuroVault). (At some point) I will be adding this option to all other templates, and of course will need clear documentation (or a function to get) the tags that are remove-able for each template. I can guarantee that the versions of the scripts that the templates have embedded will function properly, and it is at the user's own discretion to decide to use his/her version.

I tested the timing of the similarity search page, and generation of the page itself is almost instantaneous, so speeding up the page will rely almost entirely on the queries and indexing to generate the scores to go into the function, and not something from within pybraincompare. Hopefully not loading the external jquery resource, however, will speed things up a little bit! :O)
